app/src/Controller/Api/GameController.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Controller\Api;
+
+use App\Entity\Game;
+use App\GameManipulator;
+use App\Repository\GameRepository;
+use App\Repository\ShipRepository;
+use App\Twig\AppExtension;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+
+class GameController extends AbstractController
+{
+    /**
+     * Create a game and returns its hash to the player
```

Comment:
```suggestion
     * Create a game and return its hash to the player
```

or __Creates a game and returns__